middlewares/accesslog/logger_test.go

```diff
@@ -47,16 +42,15 @@ func TestLogger(t *testing.T) {
 
 	logfilePath := filepath.Join(tmp, logfileNameSuffix)
 
-	logger = NewLogger(logfilePath)
+	logger, err = NewLogHandler(logfilePath)
+	assert.NoError(t, err)
```

Comment:
We have added `testify.require` by now. Please use it here and wherever continuation of a test does not make sense anymore in case of a failed assertion.